api/api.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/LettsDoSomeCoding/ranko/logging"
+	"github.com/gorilla/mux"
+)
+
+var homePageGreeting string
+
+func StartServer(greeting string) {
+	fmt.Println("Starting server")
+
+	homePageGreeting = greeting
+	router := mux.NewRouter().StrictSlash(true)
+
+	router.HandleFunc("/", homePage)
+	router.HandleFunc("/player/{id}", createPlayerHandler).Methods("POST")
```

Comment:
So this mux package is used to define all our routing behaviour?

Reply:
Yep, that's the plan at the moment. It just seems to add a bunch of options for fancier routing and better handling of parameters over the http package's standard router.